src/iuse.cpp

```diff
@@ -5556,7 +5562,7 @@ int iuse::portal(player *p, item *it, bool, const tripoint& )
     if (it->charges < it->type->charges_to_use()) {
         return 0;
     }
-    g->m.add_trap(p->posx() + rng(-2, 2), p->posy() + rng(-2, 2), tr_portal);
+    g->m.add_trap({p->posx() + rng(-2, 2), p->posy() + rng(-2, 2), p->posz()}, tr_portal);
```

Comment:
```
ccache g++-4.7 -DRELEASE -DLOCALIZE -m64 -Os -Wall -Wextra   -Werror --std=c++11 -MMD -I/usr/include/lua5.2     -DLUA -I/usr/include/ncursesw -c src/lightmap.cpp -o obj/lightmap.o
src/iuse.cpp: In member function ‘int iuse::portal(player*, item*, bool, const tripoint&)’:
src/iuse.cpp:5565:89: error: narrowing conversion of ‘(((long int)p->player::posx()) + rng(-2l, 2l))’ from ‘long int’ to ‘int’ inside { } [-Werror=narrowing]
src/iuse.cpp:5565:89: error: narrowing conversion of ‘(((long int)p->player::posy()) + rng(-2l, 2l))’ from ‘long int’ to ‘int’ inside { } [-Werror=narrowing]
cc1plus: all warnings being treated as errors
make: *** [obj/iuse.o] Error 1
make: *** Waiting for unfinished jobs....
Build step 'Execute shell' marked build as failure
```

-- via <http://ci.narc.ro/job/Cataclysm-Matrix/Graphics=Curses,Platform=Linux_x64/3166/console>

Entertainingly, none of the other Jenkins builds complained. I still find that odd.